src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
+
+The Merge represents the official switch to using the Beacon Chain as the engine of block production. Mining will no longer be the means of producing valid blocks. Instead, the proof-of-stake validators assume this role and will be responsible for processing the validity of all transactions and proposing blocks.
+
+No history is lost. As Mainnet gets merged with the Beacon Chain, it will also merge the entire transactional history of Ethereum. You don't need to do anything. Your funds are safe.
+
+<InfoBanner>
+This transition to proof-of-stake will come with some changes to the way ether is supplied. Learn more about <a href="/upgrades/merge/issuance/">ether issuance before and after The Merge</a>.
+</InfoBanner>
+
+## What to I need to do to get ready? {#preparing-for-the-merge}
+
+The Merge is one of the most significant and anticipated upgrades in the history of Ethereum, and although in the long-term its impact will be felt by everyone, in the near-term some folks will need to take action to be fully prepared.
+
+### Users and holders {#users-holders}
+
+**You do not need to do anything to protect your funds entering The Merge.**
+
+_This bears repeating_: As a user or holder of ETH or any other digital asset on Ethereum, as well as non-node-operating stakers, **you do not need to do anything with your funds or wallet before The Merge.**
+
+Despite swapping out proof-of-work, the entire history of Ethereum since genesis remains intact and unaltered after the transition to proof-of-stake. Any funds held in your wallet before The Merge will still be accessible after The Merge. **No action is required to upgrade on your part.**
+
+As we approach The Merge of Ethereum Mainnet, **you should be on high alert for scams trying to take advantage of users during this transition.** Do not send your ETH anywhere in an attempt to "upgrade to ETH2." There is no "ETH2" token, and there is nothing more you need to do for your funds to remain safe.
+
+[More on Ethereum security](/security/#eth2-token-scam)
+
+### Node operators and dapp developers {#node-operators-dapp-developers}
+
+<ExpandableCard
+title="Staking node operators and providers"
+contentPreview="If you are a staker running your own node setup or a node infrastructure provider, there are a few things you need to be aware of to be prepared for The Merge."
+id="staking-node-operators">
+Key action items include:
+
+- Running _both_ a consensus layer client as well as an execution layer client. Third-party endpoints to obtain execution data will be unavailable after The Merge.
+- Authenticating both execution layer and consensus layer clients with a shared JWT secret so they can securely communicate with one another.
+
+Not completing the above items will result in your node being seen as "offline" after The Merge until both layers are synced and authenticated.
```

Comment:
This should be nested under the 2nd list item or else this forms two different  lists in HTML (a11y)

Reply:
Pulling this line below the list (since it applies to both the first two, don't feel it should be nested under one of them). Changing these to a numbered list, and in the "Not completing the above..." just updating to "Not completing the first two items above..."